### PR TITLE
Fixed data directory

### PR DESCRIPTION
**Pull request by bilbercode**

### CODE REVIEW

**Comment by jakekeeys** [comment]
should be using the data dir for restores too

File: run.sh [hunk 1]
```diff
@@ -60,25 +60,33 @@ function doBackup {
...
 
+    ##
+    # change directory to our data mount
+    #
+    cd data
+
     ## package up the lot into a tar.gz
     tar -zcvf ${BACKUP_NAME} ${TIMESTAMP}
-
+    rm -rf ${TIMESTAMP}
```
Comment (jakekeeys) [inline run.sh]: is it a dir or a file

Comment (bilbercode) [reply run.sh]: dir

File: run.sh [hunk 2]
```diff
@@ -60,25 +60,33 @@ function doBackup {
     ##
     # Create our mongo dump into a timestamped directory
     #
-    mongodump --uri ${MONGO} -o ${TIMESTAMP}
+    mongodump --uri ${MONGO} -o data/${TIMESTAMP}
```
Comment (jakekeeys) [inline run.sh]: why not just cd before this

File: run.sh [hunk 3]
```diff
@@ -60,25 +60,33 @@ function doBackup {
...
 
     if [[ $? -ne 0 ]];then
      echo "Failed to create mongo dump!"
         exit 1
     fi
 
+    ##
+    # change directory to our data mount
+    #
+    cd data
```
Comment (jakekeeys) [inline run.sh]: can't we just use paths rather than changing directory

**Pull request merged**